### ISSUE TITLE
Doesn't work in Waterfox 56

### ISSUE THREAD

**Issue by timur-g** [Istador/old-search-fixed#9]
I'm new to Waterfox and I use WaterfoxPortable_56.2.12_English.
I installed old-search-fixed 3.0.3 and in it's Options turned on "Search Bar" but I can't see it working. 
When I click another engine to search, default remains the same.

**Comment by Istador** [comment]
This is strange and I can't reproduce this.

I personally use 56.2.12 (64-bit) and use this add-on daily (on Windows and Linux).

I just downloaded a fresh Waterfox Windows Portable version (56.2.12), removed my existing Waterfox profile (renamed it), installed the Portable, started the Portable (a new blank profile), installed this add-on using the latest XPI from the Github release page, and it worked instantly as expected.

---

Do you have other add-ons installed that might interfere with the Search Bar?

Or do you use a custom Search Bar provided by another add-on?
This add-on only injects itself in the default bars provided by the browser.

---

Do you search via the Search Bar ("Search") or via the Address/URL Bar ("Search or enter address")?

This add-on has separate settings for both:
- The "Search Bar" option only enables it for the Search Bar.
- The "URL Bar" option only enables it for the Address/URL Bar.

When you install the add-on for the first time, both should be enabled by [default](https://github.com/Istador/old-search-fixed/blob/public/defaults/preferences/prefs.js).

If in doubt, try to enable all options of this add-on.

---

If you can't get it to work, then please check if there are any error messages logged:

On the Add-on's page in `about:addons`, there is a menu button at the top with the option to "Debug Add-ons" (`about:debugging#addons`), open it.
On the following page, please click on "Debug" for `Old Search Fixed 3`.
It should open a new window with a debug console.
Check if there are any errors present in the window or are added to it while using the Search or URL Bars.

**Comment by timur-g** [comment]
I did have other addons but I also tried with empty profile and also no luck.
Printscreen od options I tried is at https://i.imgur.com/nUjoNEs.png
I search via Search Bar, I click another engine and expect it to be remembered as a default. 
I tried debug but I don't see any Console.
It behaves like not installed at all. 
I installed from this page (downloaded as -zip, unpacked and packed contents as .zip and renamed as .xpi).
